R/scregclust.R

```diff
@@ -95,6 +95,9 @@
 #'                           target gene.
 #' @param nowarnings When turned on then no warning messages are shown.
 #' @param verbose Whether to print progress.
+#' @param quick_mode Wheter to use a reduced number of noise targets to speed up computations.
+#' @param percent A number between 0 and 1 indicating the amount of noise targets to use in 
+#'                the re-allocation process if \code{quick_mode = TRUE}.
```

Comment:
`percent` is too unspecific of a argument name. My suggestion would be to have a single parameter `quick_mode_percent`. If it is equal to 1 all proceeds as usual (this should be the default) and if it is in [0, 1) then the quick mode takes effect. When you check for `quick_mode == TRUE` you check for `percent != 1` anyways.

If you insist on having two arguments, then make the second one more specific (i.e. `quick_mode_percent`)

Reply:
`percent` changed to `quick_mode_percent` for clarity and the range of valid `quick_mode_percent` was changed from [0, 1] to [0, 1). Still kept `quick_mode` as a bool but set default to `FALSE`.